public/util/data.ts

```diff
@@ -271,6 +273,22 @@ export function fetchResultExemplars(
   return new Promise((res) => res());
 }
 
+export function minimumRouteKey(): string {
```

Comment:
It might be worth adding a comment explaining the purpose of this function.
This look like a major improvement, and the name `minimumRouteKey` is not really clear.

Reply:
Will do.